lib/internal/test_runner/utils.js

```diff
@@ -103,7 +103,18 @@ const kDefaultDestination = 'stdout';
 async function getReportersMap(reporters, destinations) {
   return SafePromiseAllReturnArrayLike(reporters, async (name, i) => {
     const destination = kBuiltinDestinations.get(destinations[i]) ?? createWriteStream(destinations[i]);
-    let reporter = await requireOrImport(kBuiltinReporters.get(name) ?? name);
+
+    // Load the test reporter passed to --test-reporter
+    const reporterSpecifier = kBuiltinReporters.get(name) ?? name;
+    let parentURL;
+    try {
+      const { pathToFileURL } = require('internal/url');
```

Comment:
This should be top-level, `internal/url` is snapshotted so lazy-loading it gives us nothing.

Reply:
I've been noticing this type of comment a fair amount lately. We should lint for this if possible because the modules that are snapshotted changes over time and I don't think everyone keeps up with what that list is.